stable/redis/values.yaml

```diff
@@ -221,7 +221,16 @@ master:
   ## Redis Master pod priorityClassName
   # priorityClassName: {}
 
-
+  ## Sysctl InitContainer
+  ## used to perform sysctl operation to modify Kernel settings (needed sometimes to avoid warnings)
+  sysctlImage:
+    enabled: false
+    command: []
+    registry: docker.io
+    repository: busybox
```

Comment:
`bitnami/minideb` doesn't have the `sysctl`command built-in, given the purpose of this initContainer we may prefer an image with this command.

However, in the specific case of the `somaxconn` setting I was trying to edit, we can actually workaround by simply using `echo 10000 > /proc/sys/net/core/somaxconn` so minideb would be fine. I'm not sure if this would work with any setting that a user may want to edit.

Let me know what you prefer.

Reply:
I'd rather use **minideb** since it's released every single day ensuring it's not affected by CVEs.

You can also use in your command:

```
install_packages systemd && sysctl ...
```

> One cool thing about using Minideb is that you can install packages.